Source/Extensions/Blazorise.DataGrid/BaseDataGridColumn.cs

```diff
@@ -1,21 +1,15 @@
 ﻿#region Using directives
+
```

Comment:
I hate these spaces here 😂. CodeMaid did it

Reply:
You're goddam picky! 😂

Reply:
Haha someone have to be. I worked on too many projects to know consistency is a must. Especially in a team